UserTools/RunControl/RunControl.cpp

```diff
@@ -15,14 +15,19 @@ RunControl::RunControl():Tool(){}
 bool RunControl::Initialise(std::string configfile, DataModel &data){
   
   InitialiseTool(data);
-  InitialiseConfiguration(configfile);
-  //m_variables.Print();
   
-  // printf("d1\n");
+  m_configfile = configfile;
+  InitialiseConfiguration(m_configfile);
+  ExportConfiguration();
```

Comment:
export should come after load config

Reply:
It should, though at the time this was written RunControl did not have a LoadConfig function.